frontend/src/app/shared/components/sidemenu/sidemenu.component.sass

```diff
@@ -53,6 +53,9 @@
     line-height: 30px
     text-decoration: none
 
+    &:has(+ span.spot-icon_enterprise-badge)
+      margin-right: 4px
```

Comment:
though I personally love the `has` selector, with Firefox being out it might not be supported well enough yet to warrant usage: https://caniuse.com/css-has